pkg/rc/fields/fields_test.go

```diff
@@ -1,37 +1,44 @@
-package fields_test
+package fields
 
 import (
 	"encoding/json"
 	"testing"
 
 	. "github.com/anthonybishopric/gotcha"
 	"github.com/square/p2/pkg/manifest"
-	"github.com/square/p2/pkg/rc/fields"
 )
 
 func TestJSONMarshal(t *testing.T) {
 	mb := manifest.NewBuilder()
 	mb.SetID("hello")
 	m := mb.GetManifest()
 
-	rc1 := fields.RC{
-		ID:       "hello",
-		Manifest: m,
+	rc1 := RC{
+		ID:              "hello",
+		Manifest:        m,
+		ReplicasDesired: 2,
 	}
 
 	b, err := json.Marshal(&rc1)
 	Assert(t).IsNil(err, "should have marshaled")
 	t.Log("serialized format:", string(b))
 
-	var rc2 fields.RC
+	var rc2 RC
 	err = json.Unmarshal(b, &rc2)
 	Assert(t).IsNil(err, "should have unmarshaled")
 	Assert(t).AreEqual(rc1.ID, rc2.ID, "RC ID changed when serialized")
 	Assert(t).AreEqual(rc1.Manifest.ID(), rc2.Manifest.ID(), "Manifest ID changed when serialized")
 }
 
 func TestZeroUnmarshal(t *testing.T) {
-	var rc fields.RC
+	var rc RC
 	err := json.Unmarshal([]byte(`{}`), &rc)
-	Assert(t).IsNil(err, "error unmarshaling")
+	if err == nil {
+		t.Errorf("expected an error unmarshaling an RC missing the replicas_desired field")
```

Comment:
That's one reason.

I also don't like those assertions in general because they take `interface{}` arguments, so when the types change they might not be comparable anymore and the compiler doesn't complain you just get random test failures.